src/history/__tests__/historyContainer.test.ts

```diff
@@ -0,0 +1,40 @@
+import * as actions from 'history/historyActions'
+import { filters } from 'history/historyConst'
+import { mapDispatchToProps, mapStateToProps } from 'history/historyContainer'
+import { GlobalState } from 'types'
+
+import { mockChartData, mockGlobalState } from './mockData'
+
+describe('history container', () => {
+  it('should map state to props correctly', () => {
+    const expectedStateProps = {
+      company: 'AAPL',
+      history: mockChartData,
+      dateFilter: filters.YTD,
+      priceFilter: filters.CLOSE
+    }
+
+    expect(mapStateToProps(mockGlobalState as GlobalState)).toEqual(expectedStateProps)
+  })
+
+  it('should map dispatch to props correctly', () => {
```

Comment:
I suggest you break this down into three separate tests all of which are described by MapDispatchToProps, and then called individually.